src/plugins/data/public/query/query_string/language_service/language_service.ts

```diff
@@ -27,10 +28,6 @@ export class LanguageService {
     this.queryEditorExtensionMap = {};
   }
 
-  public createDefaultQueryEditor() {
```

Comment:
i think the goal here is that the language service exposes public methods. So this was the intent instead,, that developers can add a new language an just call
```
const editor = data.query.queryString.getLanguageService().createDefaultQueryEditor();`
```

if they wanted to use the default editor for their language

Reply:
added a public function call within the language service: 

```
  public createDefaultLanguageReference = () => {
    return createDefaultLanguageReference();
  };
```